### PR TITLE
Add component service, and search-list component

### PR DESCRIPTION
**Pull request by L-sampson**
Users are able to search movies, and shows.

### CODE REVIEW

File: src/app/services/internal-component.service.ts
Comment (shamiguel) [inline src/app/services/internal-component.service.ts:1]: only nitpick may be changing the name to something more specific I think

**Review by shamiguel — APPROVED** [review]
LGTM

File: src/app/services/internal-component.service.ts
Comment (L-sampson) [reply src/app/services/internal-component.service.ts:1]: OK sounds good. I will change the InternalComponentService to "SearchApiService" or "MediaSearchService" to be more specific. I agree that the name could be better ;)

**Review by shamiguel — APPROVED** [review]
LGTM

**Pull request merged**